sdk/logical/storage_transactions.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2024 OpenBao a Series of LF Projects, LLC
+// SPDX-License-Identifier: MPL-2.0
+
+package logical
+
+import (
+	"context"
+)
+
+// Transactional is an optional interface for backends that support
+// interactive (mixed code & statement) transactions in a similar
+// style as Go's Database paradigm. This is equivalent to
+// physical.Transactional, not the earlier, one-shot version of the
+// interface.
+type Transactional interface {
+	// This function allows the creation of a new interactive transaction
+	// handle, only supporting read operations. Attempts to perform write
+	// operations (
```

Comment:
The explanation sentence seems to be unfinished.

Reply:
Good catch, this was meant to read:

Attempts to perform write operations (Put, Delete) will err.